src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        return new Ball(new Random().nextInt(99) + 1, color.getRandomColor());

```

Comment:
create constant MAX_VALUE = 100
```suggestion
        return new Ball(new Random().nextInt(MAX_VALUE), color.getRandomColor());

```